website/cue/reference/components/sources/base/host_metrics.cue

```diff
@@ -1,51 +1,6 @@
 package metadata
 
 base: components: sources: host_metrics: configuration: {
-	cgroups: {
```

Comment:
Noting that this diff is from `make generate-component-docs`.

Reply:
Ah, yes, this is a known issue when generating on OSX #15193 . Could you back out this diff?

Reply:
Sure thing, reverted in [38f8638](https://github.com/vectordotdev/vector/pull/15745/commits/38f86387c8f38983f1aa684fbc8d1aa617724402)